src/cmd/tools/m3db_tool/main/common/http.go

```diff
@@ -0,0 +1,115 @@
+package common
+
+import (
+	"flag"
+	"fmt"
+	"go.uber.org/zap"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+var (
+	EndPoint *string
+)
+
+// This is used across all the commands
+func init() {
+	EndPoint = flag.String("endpoint", "http://localhost:7201", "The url for target m3db backend.")
```

Comment:
Already mentioned elsewhere, these global variables should be avoided as they make the code unclear.